perfscale_regression_ci/scripts/scalability/node-affinity-anti-affinity.sh

```diff
@@ -0,0 +1,171 @@
+#/!/bin/bash
+################################################
+## Auth=prubenda@redhat.com lhorsley@redhat.com
+## Desription: This testcase tests Node Affinity and anti-affinity as we approach to node capacity	
+## Polarion test case: OCP-18082 - NextGen Pod scheduler at capacity with Node Affinity and Anti-Affinity rules	
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-18082
+## Cluster config: Cluster needed in AWS EC2 (m5.xlarge, 3 master/etcd, 3 worker/compute nodes)
+## kube-burner config: perfscale_regerssion_ci/kubeburner-object-templates/node-affinity-anti-affinity-config.yml
+################################################ 
+
+
+source ../../utils/run_workload.sh
+source ../custom_workload_env.sh
+source ../common.sh
+source node-affinity-anti-affinity_env.sh
+
+function show_node_labels() {
+  oc get node --show-labels
+  oc get node -l cpu=4
+  oc get node -l cpu=6
+  oc get node -l beta.kubernetes.io/arch=intel
+}
+
+function check_no_error_pods()
+{
+  error=`oc get pods -n $1 | grep Error | wc -l`
+  if [ $error -ne 0 ]; then
+    echo "$error pods found, exiting"
+    #loop to find logs of error pods?
+    exit 1
+  fi
+}
+
+export ANTI_AFFINITY_JOB_ITERATION=${ANTI_AFFINITY_JOB_ITERATION:-130}
+export AFFINITY_JOB_ITERATION=${AFFINITY_JOB_ITERATION:-200}
```

Comment:
I think we should be able to run this test with both of the job iterations here set to 190. That's what I've run in previous release regression runs

Reply:
done.